### DISCUSSION TITLE
知识库有考虑以后可以上传图片，根据图片中的文字内容分块化吗

### DISCUSSION THREAD

**Discussion by m0eak** [lobehub/lobehub#4595] [General Question] [answered]
知识库有考虑以后可以上传图片，根据图片中的文字内容分块化吗

**Comment by arvinxx** [comment, marked as answer]
有，知识库的能力会逐步加强，有任何需求欢迎提

**Reply by Sun-drenched** [reply]
有考虑像“GPT 学术优化”一样，引入Doc2X等第三方文档识别服务吗？

**Reply by arvinxx** [reply]
多种文档识别服务是接下来知识库模块演进的一个重点方向(用户可以在知识库级别配置文档处理服务)，不过不在近期的实现规划里，这部分估计要明年 Q1 再做了

**Reply by m0eak** [reply]
> 多种文档识别服务是接下来知识库模块演进的一个重点方向(用户可以在知识库级别配置文档处理服务)，不过不在近期的实现规划里，这部分估计要明年 Q1 再做了

有roadmap吗，图片的分块对我来说很有用
